cmd/server/main.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"context"
+	"google.golang.org/grpc"
+	"gorm.io/gorm"
+	//"google.golang.org/grpc/codes"
+	//"google.golang.org/grpc/status"
+	"log"
+	"net"
+	"os"
+
+	"github.com/jeka2708/golang-training-enterprise-grpc/pkg/api"
+	"github.com/jeka2708/golang-training-enterprise-grpc/pkg/db"
+	"time"
+)
+
+func main() {
+	var listen = os.Getenv("LISTEN")
+	if listen == "" {
```

Comment:
this logic with assigning default values in case env var doesn't exist better to move to the init method